support/mender-inventory-intervals

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+
+json_file="/etc/mender/mender.conf"
+
+if [ ! -f "$json_file" ]; then
+    echo "File not found: $json_file" >&2
+    exit 1
+fi
+
+to_kebab_case() {
+    string="$1"
+    # Remove the word "interval"
+    string=$(echo "$string" | sed 's/Interval//g')
+    # Replace capital letters with hyphen-lowercase
+    string=$(echo "$string" | sed -E 's/\([A-Z]\)/-\1/g')
+    # Convert to lowercase
+    string=$(echo "$string" | tr '[:upper:]' '[:lower:]')
+    # Preppend the "interval" word
+    string=$(echo "$string" | sed -E 's/^/interval/g')
```

Comment:
Should there be a hyphen after `interval`?

Reply:
Yes, it should look something like this:

```
$ string=RetryPollIntervalSeconds
$ string=$(echo "$string" | sed 's/Interval//g')
$ echo $string

RetryPollSeconds

$ string=$(echo "$string" | sed 's/\([A-Z]\)/-\1/g')
$ echo $string

-Retry-Poll-Seconds

$ string=$(echo "$string" | tr '[:upper:]' '[:lower:]')
$ string=$(echo "$string" | sed -E 's/^/interval/g')
$ echo "$string"

interval-retry-poll-seconds
```

Reply:
Ah, because there is dangling hyphen earlier. You're right, I missed that. Looks good then!